### ISSUE TITLE
the length of get_score is less than the length of test datas

### ISSUE THREAD

**Issue by JialeDeng** [haowen-xu/donut#21]
Hi, Haowen Xu!
  I use the method "DonutPredictor.get_score()" to get the score of  every point in test datas, only to find that the length of score is **119** less than the length of test datas, so I can not know which points are not normal. How can I get to right score? Or is there other methods to get the unnormal points? Thanks a lot, here is my code:
-------------------------------------------------------------------------------
from donut import DonutTrainer, DonutPredictor
from construct import model, model_vs
from prepare_data import train_values, train_labels, train_missing, mean, std, test_missing, test_values, values, missing, labels
import tensorflow as tf
import os

os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
trainer = DonutTrainer(model=model, model_vs=model_vs)
predictor = DonutPredictor(model)

with tf.Session().as_default():
    trainer.fit(train_values, train_labels, train_missing, mean, std)
    test_score = predictor.get_score(test_values, test_missing))
    print(test_score)

**Comment by lizeyan** [comment]
The same issue with https://github.com/haowen-xu/donut/issues/20.
Closed

**Comment by JialeDeng** [comment]
Thank you for reminding. I get to know the reason. But how can I exactly find the unnormal points using this model?

**Comment by lizeyan** [comment]
> Thank you for reminding. I get to know the reason. But how can I exactly find the unnormal points using this model?

`test_score` indicates the likelihood of normality.
Please refer to the documentation in the code:
``` python
def get_score(self, values, missing=None):
        """
        Get the `reconstruction probability` of specified KPI observations.
        The larger `reconstruction probability`, the less likely a point
        is anomaly.  You may take the negative of the score, if you want
        something to directly indicate the severity of anomaly.
        Args:
            values (np.ndarray): 1-D float32 array, the KPI observations.
            missing (np.ndarray): 1-D int32 array, the indicator of missing
                points.  If :obj:`None`, the MCMC missing data imputation
                will be disabled. (default :obj:`None`)
        Returns:
            np.ndarray: The `reconstruction probability`,
                1-D array if `last_point_only` is :obj:`True`,
                or 2-D array if `last_point_only` is :obj:`False`.
        """
...
```

**Comment by JialeDeng** [comment]
Yeah, I get that. Thank you so mcuh.
